content/docs/setup/kubernetes/multicluster-install/index.md

```diff
@@ -221,28 +232,31 @@ to the remote cluster.
     EOF
     {{< /text >}}
 
-At this point, the remote clusters' `kubeconfig` files have been created in the current directory.
-The filename for a cluster is the same as the original `kubeconfig` cluster name.
+At this point, you created the remote clusters' `kubeconfig` files in the
+current directory. The filename of the `kubeconfig` file is the same as the
+original cluster name.
 
-## Instantiate the credentials for each remote cluster
+## Instantiate the credentials for each remote cluster {#credentials}
 
-Execute this work on the cluster running the Istio control
-plane using the `WORK_DIR`, `CLUSTER_NAME`, and `NAMESPACE` environment values set during the
-[Generate kubeconfig for remote clusters](#generate-kubeconfigs-for-remote-clusters) steps.
+Perform this procedure on the cluster running the Istio control plane. This
+procedure uses the `WORK_DIR`, `CLUSTER_NAME`, and `NAMESPACE` environment
+values set and the file created for the remote cluster's secret from the
+[previous section](#kubeconfig).
 
-* _(Optional)_  Source the environment variables file created for the remote cluster secret:
+If you created the the environment variables file for the remote cluster's
+secret, source the file with the following command:
 
     {{< text bash >}}
     $ source remote_cluster_env_vars
     {{< /text >}}
 
-Istio can be installed in a different namespace other than
-istio-system.
+You can install Istio in a different namespace. This procedure uses the
+`istio-system` namespace.
 
-The local cluster running the Istio control plane does not need
-it's secrets stored and labeled. The local node is always aware of
-its Kubernetes credentials, but the local node is not aware of
-the remote nodes' credentials.
+You don't need to store and label the secrets for the local cluster running the
```

Comment:
Although I am not 100% sure, but I think you "can't" label the secrets for the local cluster.  I think things break,  but not 100% sure.  Perhaps slightly stronger language here would be good.

Reply:
@john-a-joyce ya - that would result in duplicate EDS endpoints.  May want to use a {{ warning }} here as well or a `>`

Reply:
Does this sound better:
{{ warning }} Do not store and label the secrets for the local cluster running the Istio control plane. Istio is always aware of the local cluster's Kubernetes credentials. 

The rest of the text would be outside of the warning.

Reply:
Yes - That is good.

Reply:
Added the warning.